src/index.js

```diff
@@ -193,7 +193,9 @@ Debug.extend(exported, {isSafari, getPerformanceMetrics: PerformanceUtils.getPer
  *   be dramatically worse than expected (e.g. a software WebGL renderer would be used).
  * @return {boolean}
  * @example
- * mapboxgl.supported() // = true
+ * // Show an alert if the browser does not support Mapbox GL
+ * if (!mapboxgl.supported()) {
+ * alert('Your browser does not support Mapbox GL');
```

Comment:
Show the closing `}` for this if statement:

```suggestion
 * // Show an alert if the browser does not support Mapbox GL
 * if (!mapboxgl.supported()) {
 *   alert('Your browser does not support Mapbox GL');
 * }
```

Reply:
Oh my, thanks so much for this correction @HeyStenson!